src/site/xdoc/checks/misc/translation.xml.template

```diff
@@ -93,15 +93,15 @@ messages.properties: Key 'ok' missing.
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/translation/Example1.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/translation/Example1.java"/>
           <param name="type" value="config"/>
```

Comment:
please review update https://github.com/checkstyle/checkstyle/pull/16141/files#diff-1a0e99bbb8140dd6852b3339803fbdbf9b106347f8a55ee9cc93f38f87995b7dR54

to see how to use "raw" type and put in web site content of whole file of properties.